README.md

```diff
@@ -38,11 +38,22 @@ $ export KINESIS_PARTITION_KEY_TEMPLATE={{ index .Container.Config.Labels "app"
 ### stream creation
 By default, **logspout-kinesis** doesn't create a stream if it is missing from Kinesis. You can enable this by creating an environment variable `KINESIS_STREAM_CREATION` and set it to `true`.
 
+### stream tagging
+By default, **logspout-kinesis** doesn't tag a stream it just created. You can enable this by creating an environment variable `KINESIS_TAG_STREAM` and set it to `true`.
+Then you can set the `KINESIS_STREAM_TAG_KEY` environment variable to set the tag key, and the `KINESIS_STREAM_TAG_VALUE` variable to set the template you want to use for the tag value.
+
+A complete example would be:
+```console
+KINESIS_TAG_STREAM=true
+KINESIS_STREAM_TAG_KEY="app"
```

Comment:
Yeah, I agree. Felt like a lot when I wrote it. I think reduce it to two is the right thing to do.
Thought about doing key and value in the same template but this seems crazy parsing and code wise.